lib/tdf3/src/ciphers/aes-gcm-cipher.ts

```diff
@@ -46,14 +46,14 @@ export class AesGcmCipher extends SymmetricCipher {
    * it's parts.  There is no need to process the payload.
    */
   override async encrypt(payload: Binary, key: Binary, iv: Binary): Promise<EncryptResult> {
-    const toConcat: Buffer[] = [];
+    const toConcat: Uint8Array[] = [];
     const result = await this.cryptoService.encrypt(payload, key, iv, Algorithms.AES_256_GCM);
-    toConcat.push(iv.asBuffer());
-    toConcat.push(result.payload.asBuffer());
+    toConcat.push(new Uint8Array(iv.asArrayBuffer()));
```

Comment:
how often does `encrypt` get called? will it result in any memory issues?

Reply:
Ill test that to be sure.